### ISSUE TITLE
[CAIA A11y Support]: Authenticated Experience, Profile | H1 Issues for Screenreaders 

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#58232]
### What do you need support with?

[This ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/40593) has been stalled as we have two viable options (see the [comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/40593#issuecomment-1119199305)) 

### When do you need this support by?

Whenever you can help us

### How urgent is this request?

Not urgent, we can use this help when CAIA has the time and resources

### Supporting artifacts (if any)

- [Product outline](Insert URL here between the parenthesis)
- [User flows](Insert URL here between the parenthesis)
- [Prototype or mockups](Insert URL here between the parenthesis)
- [Epics/issue that may be helpful](https://github.com/department-of-veterans-affairs/va.gov-team/issues/40593)
- Anything else y'all think is important to share! Make sure to use the github link convention so it's accessible. Please avoid pasting direct URLS.
...

### Who's on your team?

Team name:
OCTO-DE product owner: Chante Lantos-Swette
Product manager: Travis Cahill
Designer: Liz Lantz
FE engineer: Adam Whitlock
Product/team Slack channel: accountexp-authexp
...

### Do you have an accessibility champion on your team?

No, we don't have a champion (yet)

**Comment by mtcA6** [comment]
@sara-amanda let me know if you have any questions or if you want to synchronously chat through this.

**Comment by sara-amanda** [comment]
## CAIA Update
@mtcA6 Thank you for resurrecting this issue and for your focus on accessibility. 🙌  

After reviewing the information in this ticket, related [Slack thread](https://dsva.slack.com/archives/C909ZG2BB/p1651153191682419) and related tickets and EPIC, and consulting with @joshkimux, who was in the thread conversation and previous tickets, it appears that this is a `defect-4`.

- This will require a more **permanent solution**, _instead of a bandaid fix._ 
- The a11y issues appear to be a result/symptom of the root cause of the issue.

### Root Cause
I am going to recommend that we dedicate some time to work with DST and CAIA (@kristinoletmuskat looping you in here since this is a larger IA question that needs to be explored), to come up with a more permanent solution. 

Also tagging @AngelaFowler82 here, as she was in the discussion thread last year.

That being said, this is also one of those **super rare wicked problems**, and after talking to Angela I think it's worth putting this on pause until we can loop in [@mnorthuis](https://github.com/mnorthuis) for her perspective on what implications this might have for (1) sitewide IA and (2) future iterations on profile.

### OPEN & BLOCKED EPIC RELATED ISSUE
FE | Profile | Screenreader Accessibility Issue | Sections of Profile are above the H1 #40593 
> - **Background**: The Accessibility team informed us that the links to the various sections of the Profile are above the H1 making it difficult for screenreaders to interpret and read the page structure correctly. See [slack](https://dsva.slack.com/archives/C909ZG2BB/p1651153191682419) thread.
> - **Basically 2 available options** [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/40593#issuecomment-1119199305) ...

>- [This issue is one of those things where we tried some attempts, neither solution was ideal](https://github.com/department-of-veterans-affairs/va.gov-team/issues/40593#issuecomment-1377785508), and after meeting with accessibility we put it on hold.

> - **Probably warrants some further discussion and decisions made around how we would want to proceed forward.**
>   - I did have 2 'viable' solutions commented above.
>   - But the 1st solution didn't work for screen readers
>   - The second solution was a bit more of a hack, that used a hidden heading to provided screen reader only content.

## Related Tickets (Known)
_As of 5/18/2023, these are the tickets I was able to locate._ 
- **CLOSED** - [Create accessible heading component for Profile, fix a couple edge cases for personal information#21022](https://github.com/department-of-veterans-affairs/vets-website/pull/21022)
- **CLOSED** - #19552  
- **CLOSED** - #23756 
- **OPEN** - #41697 
- **OPEN** -  #40593 (mentioned above)
- **CLOSED** -  #41001 

## Additional Information Request
_Feel free to  add any other relevant information to this ticket that can be referenced, when meeting on this topic._ 

## Next Steps
@mtcA6 

- [ ] Review open and closed tickets mentioned, above. 
- [ ] @mtcA6, it would be a good idea to schedule a meeting with @kristinoletmuskat to discuss the IA piece, first.
- [ ] @kristinoletmuskat to then possibly consult with @mnorthuis who was working on this previously and could provide historical insights, as well. 
- [ ] Loop a11y back in afterwards to consult on accessibility aspect.

cc: @coforma-terry @SarahKay8

**Comment by kristinoletmuskat** [comment]
Update: talked with the profile team on 6/2/23 and I personally understood the problem a little more, but I think we need to get a brain trust together to talk through this. 

I asked @jaredcunha his opinion, and he created a ticket here to document a potential solution: https://github.com/department-of-veterans-affairs/va.gov-team/issues/59760

**Comment by kristinoletmuskat** [comment]
I really don't have much to add here! I think we can bring this as an additional point to our meeting next wednesday to talk through this with Sara, Jared, and Josh.

**Comment by sara-amanda** [comment]
Agreed, @kristinoletmuskat - let's hold until the sync next week. I am going to bump this to "blocked" status @coforma-terry and @RLHecht , due to the hubless spoke needing addressed.
